api/models.go

```diff
@@ -192,18 +192,36 @@ type ListTransactionsResponse = []Transaction
 
 // TODO: camelCase
 type Transaction struct {
-	Type            string      `json:"type"`
-	Invoice         string      `json:"invoice"`
-	Description     string      `json:"description"`
-	DescriptionHash string      `json:"description_hash"`
-	Preimage        *string     `json:"preimage"`
-	PaymentHash     string      `json:"payment_hash"`
-	Amount          uint64      `json:"amount"`
-	FeesPaid        uint64      `json:"fees_paid"`
-	CreatedAt       string      `json:"created_at"`
-	SettledAt       *string     `json:"settled_at"`
-	AppId           *uint       `json:"app_id"`
-	Metadata        interface{} `json:"metadata,omitempty"`
+	Type            string             `json:"type"`
+	Invoice         string             `json:"invoice"`
+	Description     string             `json:"description"`
+	DescriptionHash string             `json:"description_hash"`
+	Preimage        *string            `json:"preimage"`
+	PaymentHash     string             `json:"payment_hash"`
+	Amount          uint64             `json:"amount"`
+	FeesPaid        uint64             `json:"fees_paid"`
+	CreatedAt       string             `json:"created_at"`
+	SettledAt       *string            `json:"settled_at"`
+	AppId           *uint              `json:"app_id"`
+	Metadata        *lnclient.Metadata `json:"metadata,omitempty"`
+	Boostagram      *Boostagram        `json:"boostagram,omitempty"`
+}
+
+type Boostagram struct {
+	AppName        string `json:"app_name"`
```

Comment:
Most of our JSON payloads are in `camelCase` - the transaction model and now this are inconsistent with the rest of the app - any chance you could update these?